packages/pigeon/lib/src/kotlin/kotlin_generator.dart

```diff
@@ -1906,64 +1914,69 @@ if (wrapped == null) {
       }) {
         indent.writeScoped(
           'if (pigeonRegistrar.ignoreCallsToDart) {',
-          '}',
+          '} ',
           () {
             indent.format(
               '''
               callback(
                   Result.failure(
-                      $errorClassName("ignore-calls-error", "Calls to Dart are being ignored.", "")))
-              return''',
+                      $errorClassName("ignore-calls-error", "Calls to Dart are being ignored.", "")))''',
             );
           },
+          addTrailingNewline: false,
         );
         indent.writeScoped(
-          'if (pigeonRegistrar.instanceManager.containsInstance(${classMemberNamePrefix}instanceArg)) {',
-          '}',
+          'else if (pigeonRegistrar.instanceManager.containsInstance(${classMemberNamePrefix}instanceArg)) {',
+          '} ',
```

Comment:
Pretty minor nit, but I think it's better to put the spaces before the `else` rather than after the `}` since the scoped statement doesn't need it, but the `else` statement does.